classes/migration/upgrade/v3_4_0/PreflightCheckMigration.php

```diff
@@ -488,4 +495,74 @@ protected function setFallbackVersion(): ?string
         }
         return null;
     }
+
+    /**
+     * Check the contexts' contact details before upgrade
+     *
+     * @return void
+     * @throws Exception
+     */
+    protected function checkContactSetting(): void
+    {
+        $primaryLocale = DB::table('site')
+            ->select(['primary_locale'])
+            ->first()
+            ->primary_locale;
+
+        // Get all the enabled context with context name/title
+        $contexts = DB::table("{$this->getContextTable()} AS context")
```

Comment:
The curly syntax here for expressions inside strings is looking different in this method's body compared to the rest of the file. Shouldn't it be standardized? But this is for your and Alec's consideration.

Reply:
The existing curly syntax `${VARIABEL}`  has been deprecated(with 8.2+) and should be updated to `{$VARIABLE}` more details here https://php.watch/versions/8.2/$%7Bvar%7D-string-interpolation-deprecated . it will not show the deprecation message right now but better to update those.

Thanks for spotting those and I have fixed those .